static/js/components/AssignmentForm.jsx

```diff
@@ -84,8 +84,10 @@ const AssignmentForm = ({ obj_id, observingRunList }) => {
   }));
   const classes = useStyles();
 
+  // the use of integer dates leads to some upcoming runs being
+  // left out depending on the timezone
   const upcomingObservingRuns = observingRunList.filter((run) =>
-    dayjs().isBefore(dayjs(run.run_end_utc))
+    dayjs().isBefore(dayjs(run.run_end_utc).add(2, "day"))
```

Comment:
Are you sure this doesn't accidentally add runs that are too far in the future?

Reply:
Nope because of how the comparison is done. I checked and you get the day 'before' with this.